src/dbwrapper.cpp

```diff
@@ -72,6 +72,20 @@ class CBitcoinLevelDBLogger : public leveldb::Logger {
     }
 };
 
+// Mmap is only used on 64-bit Unix systems.
+constexpr bool LevelDBUsesMmap() {
+#ifdef WIN32
```

Comment:
Could use `__LP64__` macro instead? Se https://stackoverflow.com/a/682955.

Reply:
I think that macro actually refers to the data model, e.g. I believe that you would have __LLP64__ on a 64-bit LLP system.